### ISSUE TITLE
git-sync is failing

### ISSUE THREAD

**Issue by vndroid** [cdnjs/cdnjs#14194]
The content has not been updated for two months. Did you switch to a different repository address?

**Comment by renbaoshuo** [comment]
![Snipaste_2023-05-07_08-39-35](https://user-images.githubusercontent.com/47095648/236651994-a50ee0e7-de41-40dc-8219-3629ee1f7aa5.png)

![Snipaste_2023-05-07_08-39-50](https://user-images.githubusercontent.com/47095648/236651997-698287db-acda-4315-b35c-f3a9bfe6cd66.png)

It seems that the workflow named "sync" failed.

**Comment by MattIPv4** [comment]
This has been flagged to the Cloudflare team internally for investigation.

It's worth noting that this repo is no longer the source of truth for the CDN and has not been for many years now -- cdnjs serves assets from the KV store in Cloudflare workers, not a hosted version of this repo. This repo acts as a backup origin in the event KV were to fail to serve a request, and as a convenience for third-party mirrors that replicate this repo.

**Comment by vndroid** [comment]
> This has been flagged to the Cloudflare team internally for investigation.
> 
> It's worth noting that this repo is no longer the source of truth for the CDN and has not been for many years now -- cdnjs serves assets from the KV store in Cloudflare workers, not a hosted version of this repo. This repo acts as a backup origin in the event KV were to fail to serve a request, and as a convenience for third-party mirrors that replicate this repo.

Yes, because of this, many other downstream CDN is not updating properly, so I hope you can investigate the cause of the failure soon.

**Comment by vndroid** [comment]
Is the sync problem solved? But I checked and found that the latest version of the package still did not sync successfully.

**Comment by MattIPv4** [comment]
@xtuc I'm going to re-open this as sync is still failing consistently: https://github.com/cdnjs/cdnjs/actions/workflows/sync.yml

**Comment by MattIPv4** [comment]
We've updated the git-sync logic to push frequently after adding/committing a few new versions. This should allow the CI to run far more consistently. Closing this out for now, as it seems to be running relatively happily now.

**Comment by vndroid** [comment]
Unfortunately, there are still a large number of sync requests that fail, and as a result, there are still no new versions of, for example, layui and so on. https://cdnjs.com/libraries/layui
https://github.com/cdnjs/cdnjs/actions/runs/5259385096/jobs/9504882281

**Comment by MattIPv4** [comment]
> Unfortunately, there are still a large number of sync requests that fail, and as a result, there are still no new versions of, for example, layui and so on. https://cdnjs.com/libraries/layui

What is present in this repository has no impact on what is available via the CDN -- syncing failing here does not impact libraries being available. We no longer use this repo as the source of truth for the CDN, there is a newer KV-based infrastructure that backs the CDN. Thie repository purely acts as a backup, an audit log of files, and a way for CDN mirrors to get a copy of cdnjs.

**Comment by vndroid** [comment]
> > Unfortunately, there are still a large number of sync requests that fail, and as a result, there are still no new versions of, for example, layui and so on. https://cdnjs.com/libraries/layui
> 
> What is present in this repository has no impact on what is available via the CDN -- syncing failing here does not impact libraries being available. We no longer use this repo as the source of truth for the CDN, there is a newer KV-based infrastructure that backs the CDN. Thie repository purely acts as a backup, an audit log of files, and a way for CDN mirrors to get a copy of cdnjs.

That's the problem, many components are missing from this backed-up library.

**Comment by MattIPv4** [comment]
What is happening in this repository has *no* impact on what files are available at https://cdnjs.com/libraries/layui

This repository could literally be deleted, and the files at https://cdnjs.com/libraries/layui would remain the same.

**Comment by vndroid** [comment]
I know it's not affected, but as a backup library, it's not consistent with the source library, and that's the problem.
